db/migrate/20191216192846_rename_indexes.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+class RenameIndexes < ActiveRecord::Migration[5.2]
+  def up
+    institutions_indexes = ActiveRecord::Base.connection.indexes('institutions').map(&:name)
+    if institutions_indexes.include?('institutions_temp_lower_idx')
```

Comment:
`index_exists?` does not work on this index since it's based on a function (`lower((institution)::text) text_pattern_ops`) instead of a column.

Reply:
In general these `index_exists?` checks are probably overly cautious, but they don't hurt.

Reply:
Nice pickup